HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -4022,7 +4022,7 @@ Notes;
     }
 }
 
-Code "Allow Attacking from Stomp Bounce" in "Gameplay" by "Hyper"
+Code "Allow Attacking from Stomp Bounce" in "Gameplay" by "Hyper" does "Allows Sonic to attack directly after a Stomp Bounce."
```

Comment:
Change to "Allows Sonic to perform other actions after performing a stomp bounce."